src/Components/Ignitor/src/BlazorClient.cs

```diff
@@ -345,6 +347,7 @@ public async Task<bool> ConnectAsync(Uri uri, bool connectAutomatically = true)
             HubConnection = builder.Build();
             await HubConnection.StartAsync(CancellationToken);
 
+            HubConnection.On<int, string>("JS.AttachComponent", OnAttachComponent);
```

Comment:
SignalR would complain about an args mismatch when this event wasn't handled. The exception doesn't crash the app, but it's trivial to avoid it.